mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -828,6 +828,25 @@ pub fn erc20_dev_conf(contract_address: &str) -> Json {
     })
 }
 
+/// ERC20 token configuration used for dockerized tests on Sepolia
+pub fn seploia_erc20_dev_conf(contract_address: &str) -> Json {
```

Comment:
typo in 'sepolia'

Reply:
thanks. done https://github.com/KomodoPlatform/komodo-defi-framework/pull/2169/commits/5e3e0fd87e7d023b70f1689eefbf219e95622b0d